test/manage_rest.test.ts

```diff
@@ -228,4 +228,23 @@ describe("making manage REST requests", () => {
     assert.isNotNull(result);
     assert.containsAllDeepKeys(result, ["balance_id"]);
   });
+
+  it("should retrieve a list of models", async () => {
+    const { result, error } = await deepgram.manage.getAllModels(faker.string.uuid());
+
+    assert.isNull(error);
+    assert.isNotNull(result);
+    // Add your assertions for the response data here
```

Comment:
Do we want to test that a specific property is present, like we do in the other tests?

Reply:
shhhhh i hate this logic 

but yes we do :(